vscode_client/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Lorem Ipsum Text Generator Changelog
 
 All notable changes to the Lorem Ipsum Text Generator extension are documented in this file
+## 0.4.0 [2025-06-27]
```

Comment:
YOu need to add info on what was in the new version, e.g. "Java dependency added"